packages/smbclient.rb

```diff
@@ -0,0 +1,163 @@
+# Adapted from Arch Linux samba PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/samba/trunk/PKGBUILD
+
+require 'package'
+
+class Smbclient < Package
+  description 'Tools to access a servers filespace and printers via SMB'
+  homepage 'https://www.samba.org'
+  version '4.14.2'
+  license 'GPLv3'
+  compatibility 'all'
+  source_url "https://us1.samba.org/samba/ftp/stable/samba-#{version}.tar.gz"
+  source_sha256 '95651da478743f7cb407aec81287536c096e3e18bb4981dbe47ca70bf6181f96'
+
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/smbclient-4.14.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/smbclient-4.14.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/smbclient-4.14.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/smbclient-4.14.2-chromeos-x86_64.tar.xz'
+  })
+  binary_sha256({
+    aarch64: '19d88b0a7b60b5f1002ef071f3aa8c1ecefb00067e89eff653d2c234da2ff428',
+     armv7l: '19d88b0a7b60b5f1002ef071f3aa8c1ecefb00067e89eff653d2c234da2ff428',
+       i686: 'd4952770d1531c5bfc34b1e3527cad7b8b13e2c04d9c2630c9d8bd630d6dbd7a',
+     x86_64: 'f1c215ece0eae7eb36212ccaef962347b31b660403fa8b835a12b797e0786c2b'
+  })
+
+  depends_on 'avahi'
+  depends_on 'cmocka'
+  depends_on 'cmocka' => ':build'
+  depends_on 'cups'
+  depends_on 'cups' => ':build'
+  depends_on 'docbook_xsl'
+  depends_on 'docbook_xsl' => ':build'
+  depends_on 'gpgme' => ':build'
+  depends_on 'jansson'
+  depends_on 'jansson' => ':build'
+  depends_on 'ldb'
+  depends_on 'ldb' => ':build'
+  depends_on 'libarchive'
+  depends_on 'libbsd'
+  depends_on 'libbsd' => ':build'
+  depends_on 'libcap'
+  depends_on 'libcap' => ':build'
+  depends_on 'libdb'
+  depends_on 'libdb' => ':build'
+  depends_on 'libunwind'
+  depends_on 'liburing' => ':build'
+  depends_on 'linux_pam'
+  depends_on 'lmdb' => ':build'
+  depends_on 'perl_parse_yapp' => ':build'
+  depends_on 'popt'
+  depends_on 'popt' => ':build'
+  depends_on 'talloc'
+  depends_on 'tdb'
+  depends_on 'tevent'
```

Comment:
No need to include both runtime and build dependencies.  Just include the runtime dependency.

Reply:
That's quite reasonable.

Reply:
Also, symbols are not quoted.